src/metabase/models/dashboard.clj

```diff
@@ -179,19 +179,33 @@
 
 ;;; --------------------------------------------------- Revisions ----------------------------------------------------
 
+(def ^:private excluded-columns-for-dashboard-revision
+  [:id :created_at :updated_at :creator_id :points_of_interest :caveats :show_in_getting_started :entity_id
```

Comment:
`:caveats` and `:points_of_interest` are going to be removed in my PR here (they will be dissoc'd with toucan `after-select` as the model comes out of the db): https://github.com/metabase/metabase/pull/31282. Before merging this PR check that my PR has not been merged. If it has been merged, you should remove these from the line:
```suggestion
  [:id :created_at :updated_at :creator_id :show_in_getting_started :entity_id
```